Sources/TuistLoader/Models+ManifestMappers/Target+ManifestMapper.swift

```diff
@@ -128,12 +128,19 @@ extension TuistGraph.Target {
                 return []
             }
         }
-        
+
         // remove excluding
         manifest.resources?.excluding.forEach { path in
-            allResources.remove(path: AbsolutePath(path.pathString))
+            if path.pathString.suffix(2) == "**" {
+                let exclude = path.pathString.dropSuffix("/**")
+                allResources.removeAll { element in
+                    element.path.upToLastNonGlob.dirname.contains(exclude)
+                }
+            } else {
+                allResources.remove(path: AbsolutePath(path.pathString))
+            }
```

Comment:
This is specific for SPM, where we know that the glob can be only at the end, but the excluding of resources here is much more generic and applies to any project, so we should handle globs the same way we do for sources.
References to the source mapping:
- https://github.com/tuist/tuist/blob/bf9f8dd3e25432bd5f09f92778eef90810bc7760/Sources/TuistLoader/Models%2BManifestMappers/Target%2BManifestMapper.swift#L162-L182
- https://github.com/tuist/tuist/blob/bf9f8dd3e25432bd5f09f92778eef90810bc7760/Sources/TuistCore/Graph/ModelExtensions/Target%2BCore.swift#L50-L91

Reply:
ok, then there should be another ticket and PR. This is too much for me to fix for now (and there was no problem until now). So in my PR I try to fix excluding SPM resources.

Reply:
The linked issue (https://github.com/tuist/tuist/issues/3192) refers to excluding files from resources, is not specific to SPM, isn't it?

It's fine if you want to address this only for the SPM part, but then all the logic should be in the `PackageInfoMapper`, without changing the `ProjectDescription` or `TuistGraph`, otherwise people might start using it for their project with unexpected behavior

Reply:
maybe we can also add `known issues` e.g: https://developer.apple.com/documentation/xcode-release-notes/xcode-13-beta-release-notes

Reply:
in best practises is:

// Recommended
let target = Target(name: "MyFramework", sources: ["MyFramework/Sources"])
// Discouraged
let target = Target(name: "MyFramework", sources: ["MyFramework/**/*.swift"])

https://docs.tuist.io/building-at-scale/best-practices

Reply:
I agree with @danyf90 here that we should mimic the logic that we follow for sources. Providing best practices is an option, but when there's a better option we should aim for that. If it takes an extra effort that you can't commit to we are happy to support that when we can prioritize it. We can't compromise the quality of the tool.